README.md

```diff
@@ -86,22 +86,22 @@ If you want to build them from scratch, please follow the articles in the Quicks
+| [Quickstart C# .NET for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/dotnet/from-microphone) | Windows | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C# .NET Core](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/dotnetcore/from-microphone) | Windows, Linux, macOS | Demonstrates one-shot speech recognition from a microphone. |
 | [Quickstart C# UWP for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp-uwp/) | Windows | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart C# Unity (Windows or Android)](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp-unity/) | Windows, Android | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart for Android](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/java-android/) | Android  | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Java JRE](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/java-jre/) | Windows, Linux, macOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart JavaScript](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/js-browser/) | Web | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Node.js](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/js-node/) | Node.js | Demonstrates one-shot speech recognition from a file. |
-| [Quickstart Python](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/python/) | Windows, Linux, macOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Objective-C](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec-ios/) | iOS | Demonstrates one-shot speech recognition from a file with recorded speech. |
-| [Quickstart Swift](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift-ios/) | iOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Objective-C](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec-macos/) | macOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Swift](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift-macos/) | macOS | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C# Unity (Windows or Android)](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone) | Windows, Android | Demonstrates one-shot speech recognition from a microphone. |
```

Comment:
>[Quickstart C# Unity (Windows or Android)](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone) [](start = 2, length = 156)

This says it is for UNTY, but is pointing to the UWP sample.

I think this should be this one?

[https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/unity/from-microphone](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/unity/from-microphone)

Reply:
Fixed.